### PR TITLE
Add link to gh-pages

### PR DESCRIPTION
**Pull request by jamen**
I built the html locally before I knew the gh-pages existed.  Would good to link it somewhere? 

P.S. The URL on the repository just points to `webassembly.github.io`, instead of with `/spec`.  Might be a typo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ specification language.
 It also holds the WebAssembly testsuite, which tests numerous aspects of
 conformance to the spec.
 
+View the spec at [webassembly.github.io/spec](webassembly.github.io/spec/).
```
Comment (shurcooL) [inline README.md]: This is going to be a broken link. Without a scheme, it gets treated as a relative link to:

https://github.com/WebAssembly/spec/blob/master/webassembly.github.io/spec

To fix that, you should add a scheme, preferably https://.

Comment (jamen) [inline README.md]: Thanks my bad

**Comment by rossberg-chromium** [comment]
Thanks! How about changing that to "work-in-progress spec", for extra clarity?

Can you be a bit more specific about what "URL on the repository" you are referring to in your PS?

**Comment by shurcooL** [comment]
> Can you be a bit more specific about what "URL on the repository" you are referring to in your PS?

I think he's referring to the URL that's next to the repository description. See the bottom right corner of this screenshot:

image

**Comment by jamen** [comment]
Alright I updated the text, and @shurcooL is correct

**Comment by rossberg-chromium** [comment]
Thanks @jamen.

@shurcooL, I see. Seems like I don't have the necessary admin rights to change this. @lukewagner, do you have access?

**Pull request merged**